app.js

```diff
@@ -92,7 +92,7 @@ function setReq(req) {
 }
 
 const { createProxyMiddleware } = require('http-proxy-middleware')
-app.use(`${appConfig.get('contextPath')}/graphql`, cookieParser(), csrfMiddleware, (req, res, next) => {
+app.use(`${appConfig.get('contextPath')}/graphql`, cookieParser(), (req, res, next) => {
```

Comment:
add csrfMiddleware back. https://github.com/open-cluster-management/grc-ui/pull/431/files#diff-dbed998603a94ea423636bb9aa34e71e6253bb771a3d7b51311f17a741294c1eR5 should add correct token to the ui